playbooks/robusta_playbooks/pvc_snapshots.py

```diff
@@ -0,0 +1,55 @@
+from pydantic import SecretStr
+from kubernetes import client
+from robusta.api import *
+
+
+class VolumeSnapshotParams(ActionParams):
+    """
+    :var name: name of the PersistentVolumeClaim to backup
+    :var namespace: namespace of the PersistentVolumeClaim to backup
+    :example name: some_pvc_name
+    """
+
+    name: str
+    namespace: str = "default"
+
+
+@action
+def create_pvc_snapshot(event: ScheduledExecutionEvent, params: VolumeSnapshotParams):
```

Comment:
Wouldn't it be better to use ExecutionBaseEvent.
This way we can trigger it manually by the CLI or UI.
We use the ScheduledExecutionEvent data only to add the recurrence to the name. Maybe the date is enough

Reply:
Yes, I would much rather use a `PersistentVolumeClaimEvent` here. Is it possible though? Is there a way I can define a `scheduled_trigger` in the YAML with something like that?

Reply:
I don't think we support `scheduled_trigger` and `PersistentVolumeClaimEvent`
But using `ExecutionBaseEvent` will allow it to be triggered both manually and by a scheduled trigger.

Reply:
OK, switched the recurrence number with a timestamp and started using `BaseExecutionEvent`.